.github/release_template.md

```diff
@@ -47,17 +44,14 @@ __TODO__:
 * Manual tests:
    * [ ] QA: [Changelog](https://github.com/owncloud/client/blob/master/CHANGELOG.md) test
    * [ ] QA: Regression test
-* [ ] QA: If required: create a separate test plan ticket for Windows VFS testing from [VFS Template](https://github.com/owncloud/QA/blob/master/Desktop/Test_Plan_VFS.md) - add the link here
```

Comment:
Are these VFS tests no longer relevant to us? Should they be merged into this template?

CC @TheOneRing